galaxy_ng/app/tasks/deletion.py

```diff
@@ -34,8 +35,13 @@ def delete_container_distribution(instance_ids):
     log.info("Running core.general_multi_delete to delete distro and repo")
     general_multi_delete(instance_ids=instance_ids)
 
+    repository_ids = [item[0] for item in instance_ids]
+    repository_content_pks = \
```

Comment:
This isn't working and I can't figure out why. When I add `print(repository_content_pks)`, I get `<QuerySet []>` which is wrong. It shouldn't be empty.